packages/core/src/messagingProtocols/proteus/Utility/SessionHandler/SessionHandler.ts

```diff
@@ -61,26 +61,6 @@ const parseSessionId = (sessionId: string): SessionId => {
   return match.groups;
 };
 
-interface CreateSessionParams {
-  cryptoClient: CryptoClient;
-  apiClient: APIClient;
-  sessionId: string;
-  initialPrekey?: PreKey;
-}
-const createSession = async ({
-  sessionId,
-  initialPrekey,
-  cryptoClient,
-  apiClient,
-}: CreateSessionParams): Promise<void> => {
-  const {userId, clientId, domain} = parseSessionId(sessionId);
-  const prekey =
-    initialPrekey ?? (await apiClient.api.user.getClientPreKey({id: userId, domain: domain ?? ''}, clientId)).prekey;
-  const prekeyBuffer = Decoder.fromBase64(prekey.key).asBytes;
-  await cryptoClient.sessionFromPrekey(sessionId, prekeyBuffer);
-  await cryptoClient.saveSession(sessionId);
-};
-
```

Comment:
Instead of having a method that create a single session (and duplicated code) we could instead use the `createSessions` (plural) with an array of a single session. 
This way we don't branch out and have a single piece of logic for creating sessions